pmm-app/tests/pages/qanPage.js

```diff
@@ -328,4 +335,75 @@ module.exports = {
     I.waitInUrl('tz=browser&theme=dark', 30);
     I.seeInCurrentUrl(urlPart);
   },
+
+  waitForNewQANPageLoaded() {
+    I.waitForElement(this.fields.newQANPanelContent, 30);
+    I.waitForInvisible(this.fields.newQANSpinnerLocator, 30);
+  },
+
+  applyFilterNewQAN(filterName) {
+    const filterToAplly =
+      "//span//span[contains(@class, 'checkbox-container__label-text') and contains(text(), '" +
+      filterName +
+      "')]";
+    I.waitForVisible(filterToAplly, 20);
+    I.click(filterToAplly);
+  },
+
+  async getCountOfItems() {
+    return await I.grabTextFrom(this.fields.countOfItems);
+  },
+
+  verifyChangedCount(countBefore, countAfter) {
+    assert.notEqual(countAfter, countBefore, 'Data should be changed');
+  },
+
+  async verifyFiltersSection(filterSection, expectedCount) {
+    const selectedPart = this.fields.showTop5OrAllLink + filterSection + "')]";
+    let countOfFiltersInSection = await I.grabNumberOfVisibleElements(
+      selectedPart + "/parent::p/following-sibling::div[contains(@class, 'css-180k0fv')]"
```

Comment:
This a code-generated class name, it will probably not survive on the next build. Pls ask the developer to provide a custom data-qa attribute to avoid using generated classes.